virt/virtual_machines/virtual_disks/virt-creating-and-using-default-os-images.adoc

```diff
@@ -0,0 +1,13 @@
+[id="virt-creating-and-using-default-os-images"]
+= Creating and using default OS images
+include::modules/virt-document-attributes.adoc[]
+:context: virt-creating-and-using-default-os-images
+toc::[]
+
+A _default OS image_ is a bootable disk containing an operating system and all of the operating system's configuration settings, such as drivers. You use a default OS image to create bootable virtual machines with specific configurations.
+
+To use default OS images, you must install the latest version of {VirtProductName}. You then create a ProvisionedVolumeClaim (PVC), using the {product-title} Web Console to upload a default OS image to the PVC. After uploading, you create virtual machines or virtual machine templates from the uploaded images, using the wizard in the Web Console.
```

Comment:
"You then create a ProvisionedVolumeClaim (PVC), using"

->

"You then create a persistent volume claim (PVC) by using"

Reply:
"from the uploaded images, using"

->

"from the uploaded images by using"

Reply:
Fixed.